digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/IntegrationApi.scala

```diff
@@ -596,6 +636,7 @@ class IntegrationApi(val massTransitStopService: MassTransitStopService) extends
         case "tr_bogie_weight_limits" => trWeightLimitationsToApi(bogieWeightLimitService.getByMunicipality(municipalityNumber))
         case "tr_height_limits" => trHeightLimitsToApi(heightLimitService.getByMunicipality(municipalityNumber))
         case "tr_width_limits" => trWidthLimitsToApi(widthLimitService.getByMunicipality(municipalityNumber))
+        case "carrying_capacity" => carryingCapacitiesToApi(400, municipalityNumber)
```

Comment:
since the method carryingCapacitiesToApi is specific, maybe the type could be removed